EIPS/eip-2315.md

```diff
@@ -376,7 +379,16 @@ Bytecode: `0x6005565b5d5b60035e` (`PUSH1 0x05, JUMP, JUMPDEST, RETURNSUB, JUMPDE
 
 Consumed gas: `30`
 
+## Security Considerations
+
+These changes introduce new flow control instructions, so any software which does static/dynamic analysis of EVM code needs to be modified accordingly. The `RJUMPSUB` semantics are similar to `JUMP` whereas the `RETURNSUB` instruction is different, since it can 'land' on any opcode (but the possible destinations can be statically inferred).
+
+## References
```

Comment:
This section is not valid per EIP-1 and should be removed.

Reply:
Gone.